doc/source/examples/incompressible-flow/2d-lid‐driven-cavity-flow/lid‐driven-cavity-flow.rst

```diff
@@ -0,0 +1,306 @@
+    :align: center
+    :name: geometry
+
+Only the upper wall boundary moves in the x direction with a constant velocity (:math:`u = 1` m/s) while the other boundaries are static. We will discuss later in this example the meaning of the ID of the boundary conditions used in this figure.
+
+We first investigate this case at a Reynolds number of 400 for which a steady-state solution can be easily obtained. Later, we will increase this Reynolds number to 7500 and leverage some of the advanced functionnalities of Lethe to reach a steady-state solution. 
+
+
+
+Parameter file
+--------------
+
+Lethe simulations are controlled by *parameter files* which possess the extension ``.prm``. This is the default text format of the ParameterHandler class of the deal.ii library from which Lethe derives. For more information on this class, we refer to the `deal.II documentation <https://www.dealii.org/current/doxygen/deal.II/classParameterHandler.html>`_. 
+
+Parameter files are made of subsections which describe a portion of the simulation (e.g. mesh generation, physical properties, simulation control). In parameter files, line starting with ``#`` are comments. Parameters are set using the following syntax:
+
+.. code-block:: text
+
+    set parameter name = value
+
+The syntax is flexible. Parameters do not need to be specified in a specific order, but only within the subsection in which they belong. For a full list of the parameters within Lethe, we refer to the parameter page.
+
+To set-up the lid-driven cavity case, we first need to establish the mesh used for the simulation.
+
+
+Mesh
+~~~~~
+
+The ``mesh`` subsection specifies the computational grid:
+
+.. code-block:: text
+
+    subsection mesh
+        set type                 = dealii
+        set grid type            = hyper_cube
+        set grid arguments       = 0 : 1 : true
+        set initial refinement   = 6
+    end
+
+The ``type`` specifies the mesh format used. At the moment, Lethe supports two mesh format: ``dealii`` and ``gmsh``. ``dealii`` meshes are in-situ generated meshes for simple geometries. The type of grid generated is specified by the ``grid type`` parameters and this grid is parametrized by it's ``grid arguments``. We refer to the documentation of the deal.ii `GridGenerator <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html>`_ for a detailed explanation of the available grids. 
+
+Since the lid-driven cavity problem domain is a square, we use the *hyper_cube* ``grid_type``. The arguments of this grid type are the position of the bottom left corner, the position of the top right corner and the option to colorize the boundaries in order to give each of them a unique ID. The IDs will be used to set the boundary conditions on specific part of the boundary of the domain. The ID given to each face were given in the graphical description of the case. If ``colorize`` option were set to false, all boundaries would have been given the ID ``0``.
```

Comment:
```suggestion
Since the lid-driven cavity problem domain is a square, we use the *hyper_cube* ``grid_type``. The arguments of this grid type are the position of the bottom left corner, the position of the top right corner and the option to colorize the boundaries in order to give each of them a unique ID. The IDs will be used to set the boundary conditions on specific parts of the boundary of the domain. The ID given to each face was given in the graphical description of the case. If ``colorize`` option were set to false, all boundaries would have been given the ID ``0``.
```